docs/interpreter/spark.md

```diff
@@ -262,3 +262,27 @@ select * from ${table=defaultTableName} where text like '%${search}%'
 ```
 
 To learn more about dynamic form, checkout [Dynamic Form](../manual/dynamicform.html).
+
+## Setting up Zeppelin with Kerberos
+Logical setup with Zeppelin, Kerberos Distribution Center (KDC), and Spark on YARN:
+
+<img src="../assets/themes/zeppelin/img/docs-img/kdc_zeppelin.png">
+
+####Configuration Setup
+
+1. On the server that Zeppelin is installed, install Kerberos client modules and configuration, krb5.conf.
+This is to make the server communicate with KDC.
+
+2. Set SPARK\_HOME in [ZEPPELIN\_HOME]/conf/zeppelin-env.sh to use spark-submit
+( Additionally, you might have to set “export HADOOP\_CONF\_DIR=/etc/hadoop/conf” )
+
+3. Add the two properties below to spark configuration ([SPARK_HOME]/conf/spark-defaults.conf):
```

Comment:
@felixcheung I don't know why, but that `SPARK_HOME` is shown properly. 
![screen shot 2016-02-15 at 1 02 53 pm](https://cloud.githubusercontent.com/assets/10060731/13039609/7b296178-d3e4-11e5-9c1c-22ef3f82c938.png)